package.json

```diff
@@ -129,6 +129,7 @@
     "@reach/popover": "^0.16.2",
     "@reach/portal": "^0.16.2",
     "@reach/tabs": "^0.16.4",
+    "@reach/tooltip": "^0.17.0",
```

Comment:
Adds `@reach/tooltip`. We've used this library in `@hashicorp/react-tabs` and it seems okay; it was also mentioned in the design spec.